shell/client/shell.html

```diff
@@ -441,6 +442,33 @@
   </div>
 </template>
 
+<template name="adminSettings">
+  <div id="topbar">{{> homeLink}} Admin settings</div>
+
+  <div id="admin-settings" class="main-content">
+    <div class="centered-box">
+      <form id="admin-settings-form">
+        <label title="This setting enables Google OAuth accounts for this server">Enable Google Login <input type="checkbox" name="googleLogin" value="value" checked={{googleEnabled}} /></label>
```

Comment:
These tooltips seem not very discoverable. Could we just put the description of each setting directly under it in smaller text?